### DISCUSSION TITLE
for me getting deprecation error while using cloud_firestore pacakge

### DISCUSSION THREAD

**Discussion by roshan-pc01** [firebase/flutterfire#12188] [Q&A]
cloud_firestore-4.14.0\android\src\main\java\io\flutter\plugins\firebase\firestore\FlutterFirebaseFirestorePlugin.java uses or overrides a deprecated API.

**Comment by brunos0** [comment]
I have face the same warring at 4.15.6. Only for now, reverted to 4.13.6 for now.
